src/lib/appContainer.ts

```diff
@@ -83,6 +83,9 @@ export const getAppContainer = (overrides?: IAppContainer): IAppContainer => {
     get NextAuthUserSessionGateway() {
       return new NextAuthUserSessionGateway();
     },
```

Comment:
Can we remove this implementation detail here and update the codebase to use `sessionGateway` anywhere where we have previously used `NextAuthUserSessionGateway`?